bot/plotting/random_plot_generator.py

```diff
@@ -95,37 +96,55 @@ def random_plot_generator(
                     "SEI": sei,
                 })
 
-            models = [
-                pybamm.lithium_ion.DFN(
-                    options=options
-                ),
-                pybamm.lithium_ion.SPM(
-                    options=options
-                ),
-                pybamm.lithium_ion.SPMe(
-                    options=options
-                ),
+            solvers = [
+                pybamm.CasadiSolver(mode="safe"),
+                pybamm.CasadiSolver(mode="fast with events")
             ]
 
-            model = random.choice(models)
+            solver = random.choice(solvers)
+
+            choice = random.randint(0, 3)
```

Comment:
Using `choice` as an integer to distinguish between cases is not very informative. Are we set on 2 cases or we might get an arbitrary number in the future?

Reply:
I think we might expand it in the future, I have added comments (pushing them) for what will happen if a possible random integer is selected, should I make it into a list with all the choices as elements and choose one of them randomly?